docs/source/llms/deployments/index.rst

```diff
@@ -4,24 +4,31 @@
 MLflow Deployments Server (Experimental)
 ========================================
 
-The MLflow AI Gateway service is a powerful tool designed to streamline the usage and management of
+.. important::
+   The feature previously known as **MLflow AI Gateway** in experimental status has been moved to
+   utilize the **MLflow deployments API**. This major update involves changes to API endpoints and
+   standardization for Large Language Models, both custom and SaaS-based. Users currently utilizing
+   MLflow AI Gateway should refer to the new documentation for migration guidelines and familiarize
+   themselves with the updated API structure.
```

Comment:
@BenWilson2 for addressing https://github.com/mlflow/mlflow/pull/10522#discussion_r1410108654